images/openvswitch/Dockerfile

```diff
@@ -3,16 +3,12 @@
 #
 # The standard name for this image is openshift/openvswitch
 #
-FROM openshift/origin-base
+FROM openshift/node
 
 COPY scripts/* /usr/local/bin/
 
-RUN curl -L -o /etc/yum.repos.d/origin-next-epel-7.repo https://copr.fedoraproject.org/coprs/maxamillion/origin-next/repo/epel-7/maxamillion-origin-next-epel-7.repo && \
-    INSTALL_PKGS="openvswitch" && \
-    yum install -y $INSTALL_PKGS && \
-    rpm -V $INSTALL_PKGS && \
-    yum clean all && \
-    chmod +x /usr/local/bin/*
+RUN INSTALL_PKGS="openvswitch" && \
```

Comment:
Also, +1 to switching the base of openvswitch to be the node. If you need openvswitch image you need the node image so may as well just make it the same with different entrypoint.